pyproject.toml

```diff
@@ -61,6 +61,12 @@ setup_raw_data = "thaw_slump_segmentation.scripts.setup_raw_data:setup_raw_data"
 train = "thaw_slump_segmentation.scripts.train:main"
 thaw-slump-segmentation = "thaw_slump_segmentation.main:cli"
 
+[project.optional-dependencies]
+cuda = [
+    "cupy-cuda12x>=13.3.0",
```

Comment:
is it possible to have it more flexible?

I have cuda 11.8 for example, can you check the general requirements for pythorch and cuda?

Reply:
+1, albedo has 11.7 + 12.0